Assets/MixedRealityToolkit/Common/Shaders/Standard.shader

```diff
@@ -0,0 +1,620 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+Shader "MixedRealityToolkit/Standard"
```

Comment:
Might be a little pedantic, but I'm not sure about the use of the name "Standard".  Naming could be easily confused with Unity's own standard shaders or other provided in other assets.
I know it's in the MixedRealityToolkit shader folder to distinguish from other shaders, just not sure on the name.
So this is merely a descriptional issue. Happy to be overruled.

Reply:
This is something I bounced around my team for awhile on picking a name. They shared the same concern but decided it would be best to guide users who were accustomed to using the Unity/Standard shader towards using this (purely based on name association). But, if there is another name the community would like I'm happy to change it!